getting-started-service-registry/README.adoc

```diff
@@ -0,0 +1,197 @@
+[id="chap-getting-started-service-registry"]
+= Getting Started with {product-long}
+ifdef::context[:parent-context: {context}]
+:context: getting-started-service-registry
+
+////
+START GENERATED ATTRIBUTES
+WARNING: This content is generated by running npm --prefix .build run generate:attributes
+////
+
+
+:community:
+:imagesdir: ./images
+:product-long: Red Hat Openshift Service Registry
+:product: Service Registry
+// Placeholder URL, when we get a HOST UI for the service we can put it here properly
+:service-url: https://cloud.redhat.com/beta/application-services/sr/
```

Comment:
Suggest: 
:service-url: https://cloud.redhat.com/beta/application-services/service-registry/

(The beta URL changed this week. https://cloud.redhat.com/beta/application-services/sr/ gives a 404 - see https://issues.redhat.com/browse/MGDSTRM-5144)